numpy/ma/tests/test_core.py

```diff
@@ -1723,6 +1723,13 @@ def setUp(self):
         xm[2] = masked
         self.intdata = (x, y, xm)
         self.floatdata = (x.astype(float), y.astype(float), xm.astype(float))
+        self.othertypes = np.typecodes['AllInteger'] + np.typecodes['AllFloat']
+        self.othertypes = [np.dtype(_).type for _ in self.othertypes]
```

Comment:
This line should ensure that they are not typestrings, but actual types (i.e `numpy.uint8`).